planetiler-core/src/main/java/com/onthegomap/planetiler/ForwardingProfile.java

```diff
@@ -81,13 +81,21 @@ protected ForwardingProfile(Handler... handlers) {
   }
 
   private boolean caresAboutLayer(String layer) {
-    return (onlyLayers.isEmpty() || onlyLayers.contains(layer)) && !excludeLayers.contains(layer);
+    if (excludeLayers.contains(layer) || excludeLayers.stream().anyMatch(excluded -> dependsOnLayer(layer, excluded))) {
```

Comment:
I think this logic means exclude a layer if it is explicitly excluded, or if it depends on a layer that we exclude right? This means if you exclude transportation, it will exclude transportation_name as well.  The existing behavior is that if transportation_name is used (either via include or lack of exclude) then it forces transportation to be included. Either interpretation seems valid to me, but this is a change from the current behavior.

Reply:
OK, adjusted to fix the bug (include `transportation` when `transportation_name` is used) but maintain old behavior (e.g. force `transportation` when needed even over exclude).